MagneticField/ParametrizedEngine/src/BFit3D_data.h

```diff
@@ -3,1111 +3,1210 @@
+      {3.36211317035053e-09, 1.25658042017062e-08, 7.62641420697609e-09, -1.19496539809704e-08},
+      {4.47893633875920e-11, 1.75819988726204e-11, 2.72354530778695e-11, 3.69666623833919e-11},
+      {1.16042088578226e-11, 2.30327147188395e-11, 2.40988577769171e-11, 2.81562694673083e-11},
+      {4.74697101263625e-13, -2.90369117495639e-12, -1.04669098505037e-11, -4.02425491225723e-12},
+      {-7.05380816954299e-12, -8.72596917408400e-12, 3.19771515330292e-12, -4.40050134706384e-12},
+      {-4.98787998286558e-13, -7.71851319627889e-12, -7.52987584117380e-12, -5.93988935137662e-12},
+      {6.17003212711141e-12, 1.32852523080089e-11, 6.26297194458678e-12, 4.56273278331287e-12},
+      {2.93554157726811e-11, 4.15541885327362e-11, 4.57232733413095e-11, 4.62722869397072e-11},
+      {-1.52568950995834e-11, -1.31247926749432e-11, -1.43517300660898e-11, -1.45583866494333e-11},
+      {1.15212110105309e-12, -9.15791241481853e-12, 2.26658088913781e-12, 4.34267611483039e-12},
+      {5.15536331598143e-12, 3.21260408073544e-12, 6.73524305356601e-13, 7.29773939833143e-13},
+      {9.22384175601550e-12, 2.03886324874580e-11, 5.42347288546249e-12, 1.57232829310959e-11},
+      {1.67012646975897e-11, 1.76581909961175e-11, 3.61634665223998e-11, 1.05412758668194e-11},
+      {7.10975444520234e-12, 8.92816548346090e-12, 5.81851783344833e-12, 9.69707525645154e-12},
+      {1.25815845674167e-11, 1.18856112769930e-11, 8.26546122372572e-12, 3.97100125514626e-12},
+      {4.17820164199533e-11, 4.16627531543611e-11, 6.52885520741111e-11, 4.53374973527439e-11},
+      {1.73823842674353e-11, 2.63417686630075e-11, 4.60236890531115e-12, 3.77088911611828e-11},
+      {-2.33853235804367e-12, -2.52982705982476e-11, -2.60134171748099e-11, -1.05396193521933e-12},
+      {6.98389305492476e-12, 4.42448791669421e-11, 3.08329376774030e-11, 1.13952675974324e-11},
+      {-5.26801459319174e-13, 1.38999348330103e-11, -5.66252989208178e-11, 1.09847440754115e-11},
+      {1.88840127765640e-11, -2.24470783969370e-11, 3.31682748269309e-11, 3.42423934762337e-11},
+      {-4.78351964756391e-12, 3.58598537370872e-11, -2.32915342243996e-12, -1.20018520371952e-11},
+      {-1.68497739923062e-10, -1.00503993613529e-10, -2.11541727577839e-10, -1.77303101690709e-10},
+      {-1.13709973948242e-10, -1.87076499253913e-10, -1.02555791467195e-10, -1.57080153597668e-10},
+      {-2.57959931306827e-10, -2.49133384986246e-10, -3.80886951474312e-10, -2.87301940101725e-10},
+      {1.75812685080642e-11, 1.03024336662449e-11, 2.11704116598328e-11, 4.43535221183674e-11},
+      {2.37799866461959e-10, 1.78516029373177e-10, 2.80785998479443e-10, 2.70602177412153e-10},
+      {1.34237684985945e-10, 2.53064579553359e-11, 1.50183540218717e-10, 3.71381437949006e-11},
+      {3.36871860756778e-10, 2.87290939807854e-10, 6.86524917689277e-10, 1.17431905499043e-09},
+      {3.66460831436455e-10, 3.66607928682510e-10, 4.46409858543281e-10, 6.58153206569430e-10},
+      {2.96805661632454e-10, -9.447074
```

Comment:
(Even more) Definitely the previous version was much more readable.
This is probably a god place where to forbid clang formatting.